src/libs/Navigation/Navigation.js

```diff
@@ -3,10 +3,18 @@ import React from 'react';
 import {StackActions, DrawerActions} from '@react-navigation/native';
 import {getIsDrawerOpenFromState} from '@react-navigation/drawer';
 
+import Onyx from 'react-native-onyx';
 import linkTo from './linkTo';
 import ROUTES from '../../ROUTES';
 import SCREENS from '../../SCREENS';
 import CustomActions from './CustomActions';
+import ONYXKEYS from '../../ONYXKEYS';
+
+let isLoggedIn = false;
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    callback: val => isLoggedIn = !_.isEmpty(val),
```

Comment:
Probably should be `authToken` not just seeing if there's a session object since other stuff is stored there (or could be in the future)